src/main/java/core/basesyntax/DataRecord.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class DataRecord {
+    private int supply;
+    private int buy;
+
+    public String getStringSupply() {
+        return String.valueOf(supply);
+    }
```

Comment:
The method name 'getStringSupply' does not follow the Java naming convention. Consider renaming it to something like 'getSupplyAsString' to better reflect the action it performs.